seagull/lifeforms/wiki.py

```diff
@@ -120,25 +122,70 @@ def _get_metadata(lines: List[str]) -> Dict:
         - author   : from line starting with "!Author: "
         - comments : for all other comments
     """
-    meta = {'comments':[]}
+    meta = {"comments": []}
     for line in lines:
-        if line and line.startswith("!"):
+        if line.startswith("!"):
             # parsing commented lines
             # @TODO: prettify? or delete this todo
             if line.startswith("!Name: "):
-                meta['name'] = line[len("!Name: ") :]
+                meta["name"] = line[len("!Name: ") :]
             elif line.startswith("!Author: "):
-                meta['author'] = line[len("!Author: ") :]
+                meta["author"] = line[len("!Author: ") :]
+            else:
+                meta["comments"].append(line[1:])
+        elif line.startswith("#"):
+            # parsing commented lines
+            # @TODO: prettify? or delete this todo
+            if line.startswith("#N "):
+                meta["name"] = line[len("#N ") :]
+            elif line.startswith("#O "):
+                meta["author"] = line[len("#O ") :]
+            elif line.startswith(("#R ", "#P ", "#r ")):
+                # @TODO: decide whether add R, P, r parsing?
+                raise NotImplementedError("#R, #r , #P tags not implemented")
             else:
-                meta['comments'].append(line[1:])
+                meta["comments"].append(
+                    line[3:]
+                )  # 3: for proper "#C <comment>" format
         else:
             # not a comment line
             pass
 
-    meta['comments'] = "\n".join(meta['comments'])
+    meta["comments"] = "\n".join(meta["comments"])
 
     return meta
 
+
+def _load_file_of_url(path: str) -> str:
+    """Detects if path is local or URL, loads file content
+
+    Args:
+        path (str): [description]
+
+    Returns:
+        str: [description]
+    """
+    if isfile(path):
+        logger.trace(f"reading from file [{path}]..", end="")
+        with open(path, "r") as f:
+            content = f.read()
+        logger.trace("ok")
+    elif urlparse(path).scheme in {"ftp", "http", "https"}:
+        # web-hosted file?
+        logger.trace(f"trying to download [{path}]..", end="")
+        req = urlopen(path)
+        if req.getcode() != 200:
+            raise ValueError(
+                "Invalid input URL" f" request returned {req.getcode()}"
```

Comment:
You can put this as one string:

```suggestion
                f"Invalid input URL: request returned {req.getcode()}"
```